src/openrct2/drawing/Drawing.cpp

```diff
@@ -84,19 +59,19 @@ uint8_t PaletteMap::Blend(uint8_t src, uint8_t dst) const
     assert(src != 0 && (src - 1) < _numMaps);
```

Comment:
These asserts will need to be guarded with `#ifdef _DEBUG`, as these variables are no longer compiled for non-debug builds. (Oddly enough, asserts are no-op for non-debug builds, anyway...)

Reply:
Yeah, I was hoping the macro evaluates to nothing which it does for most compilers, but looks like it needs extra care.